FF1Blazorizer/Pages/Randomize.razor

```diff
@@ -514,6 +514,11 @@
                         <TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="RandomArmorBonus" @bind-Value="Flags.RandomArmorBonus">Random Armor Bonus</TriStateCheckBox>
                         <CheckBox UpdateToolTip="@UpdateToolTipID" Id="fixMissingBattleRngEntry" @bind-Checked="Flags.FixMissingBattleRngEntry">Fix Missing Battle RNG Entry</CheckBox>
                         <TriStateCheckBox UpdateToolTip="@UpdateToolTipID" Id="swolePiratesCheckBox" @bind-Value="Flags.SwolePirates">Buffed Pirates</TriStateCheckBox>
+                        <CheckBox UpdateToolTip="@UpdateToolTipID" Id="EnablePoolParty" @bind-Checked="Flags.EnablePoolParty">Select Party From Pool of Random Characters</CheckBox>
```

Comment:
is there a reason some of these are not tri state and other are?

it seems like there isnt a reason to not have some of these as tri-state

Reply:
I'm just thinking that you wouldn't want to tri-state a specific feature, but you would for its options, but that might not be how other use tri-state.

Reply:
Some people like torturing themselves (also sometimes things like this are used in the challenge weeklies so people dont know going in, but find out right away when they start the game if something is on or not)